File: openapi-common-sdk-demo/net/MCatCloud/MCatCloud/Common/Credential.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MCatCloud.Common
{
    public class Credential
    {
        /// <summary>
        /// SecretId, can only be obtained from Tencent Cloud Management Console.
        /// </summary>
        public string SecretId { get; set; }

        /// <summary>
        /// SecretKey, can only be obtained from Tencent Cloud Management Console.
        /// </summary>
        public string SecretKey { get; set; }

        /// <summary>
        /// Token
        /// </summary>
        public string Token { get; set; }
    }
}

File: openapi-common-sdk-demo/java/MCatCloud/src/Common/Credential.java
package Common;


public class Credential {

    private String secretId;

    private String secretKey;

    private String token;

    public String getSecretId() {
        return secretId;
    }

    public void setSecretId(String secretId) {
        this.secretId = secretId;
    }

    public String getSecretKey() {
        return secretKey;
    }

    public void setSecretKey(String secretKey) {
        this.secretKey = secretKey;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }


    public Credential(String secretId, String secretKey) {
        this.secretId = secretId;
        this.secretKey = secretKey;
    }

    public Credential(String secretId, String secretKey, String token) {
        this.secretId = secretId;
        this.secretKey = secretKey;
        this.token = token;
    }
}
